### ISSUE TITLE
2.4.13 Focus Appearance: Contrast between same pixels when in-line

### ISSUE THREAD

**Issue by danhendersonede** [w3c/wcag#3225]
In the normative language for 2.4.13 Focus Appearance (Level AAA), it states a focus indicator must have
> a contrast ratio of at least 3:1 between the same pixels in the focused and unfocused states.

Would the following focus indicator fail this success criterion when used inline? 

**Focused state:**
It has a 4px offset, and the indicator itself is a solid 4px outline surrounding the interactive element.
It's total width (8px) means it overlaps the near by characters which are the same color as the indicator.

<img width="488" alt="The f from the previous word and w from the next word after the focus link are the same color as the focus indicator" src="https://github.com/w3c/wcag/assets/82847291/76247b21-25db-4000-ad33-d0f2bcd5b65a">

**Unfocused state:**
<img width="457" alt="an unfocused link called Requirements for WCAG 2.1" src="https://github.com/w3c/wcag/assets/82847291/902ea935-c651-4f31-96f3-980e49e1518e">

This focus indicator is the best our organization as agreed on to date. 

I wonder if a very partial overlap would really effect the indicator's ability to be perceived, and if there should be some sort of exception for in-line content?

Even if we were to remove the 4px offset to fix the overlapping characters issues, it would fail against punctuation directly next to a link when they are the same color, for example:

**Unfocused state:**
<img width="201" alt="A link called Full Pages directly followed by a comma" src="https://github.com/w3c/wcag/assets/82847291/019fb195-9e13-427a-a168-f872114d4860">

**Focused state:**
<img width="227" alt="The link is focused but the comma cannot be seen and the focus indicator is the same color" src="https://github.com/w3c/wcag/assets/82847291/bb1c09cf-a323-4ba6-beeb-59f28ffa4c32">

Thanks

**Comment by alastc** [comment]
Hi @danhendersonede,

The normative text includes "an area of the focus indicator meets all the following:".

A 4px outline is double what is required by the SC, so overlapping a few text characters is no problem.
